viz-lib/src/visualizations/choropleth/maps/convert-projection.js

```diff
@@ -0,0 +1,40 @@
+// This helper converts USA map from Mercator projection to Albers (USA)
+// Usage: `node convert-projection.js > usa-albers.geo.json`
```

Comment:
It would be interesting to have a way to differ those script files (extension, folder or name pattern), nothing nice to apply here in my mind now though.

Reply:
It's actually not necessarily should be reviewed as it's not a direct part of the project. I'll give some context to explain this. So, USA map usually uses an `Albers USA` projection (the one you can see here - when Alaska and Hawaii are placed in the bottom left corner). That projection, in opposite to, say, Mercator (which Leaflet uses my default) is not defined on the whole range of coordinates (-180..180/-90..90). This is also not a problem usually, but in our case visualization became crazy when tried to convert non-valid coordinates (computed an infinite zoom and entire map became 0 by 0 size). Since we don't display other layers, I decided just to convert the map from Mercator to Albers (that is - physically move Alaska and Hawaii to the place you see) and that's what this script does. Also, there is an original map - for the time when we'll find a proper way to do this.

Reply:
Sure, I realize it's not a direct part of the project, but it would be great to have something indicating this (name it `convert-project.script.js`, a `bin` folder or whatever). Not a must as I also didn't find a pattern that I liked, it's more in case anything pops up into your mind 🙂

Reply:
Eventually I'm gonna remove it, just decided to keep it for now for the reference